ui/src/app/machines/views/MachineDetails/MachineNetwork/NetworkFields/NetworkFields.tsx

```diff
@@ -26,17 +26,50 @@ export type NetworkValues = {
   vlan: NetworkInterface["vlan_id"];
 };
 
+/*
+
+
+ */
+
+/**
+ * Formik values eventually resolve to the correct types,
+ * meanwile we need to force the subnet value to be a number.
+ */
+const toNumber = (value: string | number): number | undefined => {
+  if (typeof value === "number") {
```

Comment:
If the value is `NaN` then you won't hit the next block on 42 as `typeof NaN === "number"`

Reply:
Thanks! I've swapped these around so that it checks for `string` instead.